actors/builtin/methods.go

```diff
@@ -75,25 +75,23 @@ var MethodsPower = struct {
 	OnSectorTemporaryFaultEffectiveBegin abi.MethodNum
 	OnSectorTemporaryFaultEffectiveEnd   abi.MethodNum
 	OnSectorModifyWeightDesc             abi.MethodNum
-	OnMinerSurprisePoStSuccess           abi.MethodNum
-	OnMinerSurprisePoStFailure           abi.MethodNum
+	OnMinerFallbackPoStSuccess           abi.MethodNum
+	OnMinerFallbackPoStFailure           abi.MethodNum
```

Comment:
👍 note that my current assumption is that it will be required by all miners even those regularly winning elections, since direction right now is to separate consensus considerations from these.

Reply:
Yeah, I think fallback is no longer a good name. I've been using `WindowedPoSt` a bunch, but i'm not picky about the names. I figure the word `Fallback` isnt used anywhere else, and a single global sed command can fix it all once we decide

Reply:
I like "windowed"